doc/releases/release-notes-3.2.rst

```diff
@@ -252,6 +267,8 @@ Drivers and Sensors
 
 * Clock Control
 
+  * Added driver for GigaDevice GD32 SoCs
```

Comment:
```suggestion
  * Added driver for GigaDevice GD32 SoCs (peripheral clocks configuration only)
```

Reply:
Add extra description to notify user, there have a little different than others clock control driver.